py-rse/py-scripting.Rmd

```diff
@@ -643,6 +643,35 @@ if __name__ == '__main__':
     main(args)
 ```
 
+Any Python source file can be imported by any other.
+This is why Python files should be named using [`snake_case`][snake-case]
+instead of [`kebab-case`][kebab-case]
+(an expression like `import some-thing` isn't allowed
+because `some-thing` isn't a legal variable name.)
+When a file is imported,
+the statements in it are executed as it loads.
+Variables, functions, and items defined in the file are
+then available as `module.name`, where `module` is the
+filename (without the `.py` extension) and `name` is the
+name of the item.
```

Comment:
Can this content on snake case and kebab case (which I think is worth mentioning) be a callout with the following title?

> **Module naming conventions**
>
> Any Python source file...